docs/apis/avalanchego/apis/x-chain.md

```diff
@@ -1144,6 +1149,7 @@ An output contains:
   `FvwEAhmxKfeiG8SnEvq42hc6whRyY3EFYAvebMqDNDGCgxN5Z`.)
 - `fxID`: The ID of the FX this output uses.
 - `output`: The FX-specific contents of this output.
+- `id`: The transaction ID
```

Comment:
An output does not include a transaction ID

Reply:
fixed 1fa6ed428ac455d0293f6cd97289fd50fb6764e2